accelerator/conn_mqtt.c

```diff
@@ -16,9 +16,15 @@ int main(int argc, char *argv[]) {
   struct mosquitto *mosq = NULL;
 
   // Initialize logger
+#ifdef DEBUG
```

Comment:
Shall we reply on `NDEBUG` rather than `DEBUG`?
Reference: https://stackoverflow.com/questions/2290509/debug-vs-ndebug